src/js/options.js

```diff
@@ -26,8 +26,11 @@ window.onload = () => {
         customBackground.style.display = "block";
     }
     else {
-        warning.innerHTML = "Please login for some useful features";
+        warning.innerHTML = "Please login for some useful features <button type='button' id='loginbutton' class='btn btn-success' style='background-color:green !important;color:white !importa'>Login</button>";
```

Comment:
@sk9331657 is `!important` needed here?

Reply:
@hkedia321  To override already present css.

Reply:
@sk9331657 It is not needed here. The inline CSS already override the already present CSS in the file. You can remove it.

Reply:
Done @hkedia321  . Ready to be merged.